packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -35,6 +35,7 @@ import logger from './logger'
  * @property {string} dirId - Id of the parent directory.
  * @property {boolean} name - Name of the created directory.
  * @property {boolean} executable - Indicates whether the file will be executable.
+ * @property {object} [metadata] io.cozy.files.metadata to attach to the file
```

Comment:
nit: I try to talk about file as opposed to folder. And if we're talking about both, in this case I'm trying to talk about io.cozy.files. Since here we are in a DirectoryAttribute, I would have use `folder` instead of file.